release/src/Main.hs

```diff
@@ -53,11 +55,20 @@ main = do
       if getPerformUpload upload
           then do
               $logInfo "Make release"
-              releaseToBintray upload releaseDir (map (\(a, (_, outp)) -> (a, outp)) files)
+--              releaseToBintray upload releaseDir (map (\(a, (_, outp)) -> (a, outp)) files)
```

Comment:
I thought the plan was to keep uploading to Bintray as well? If we want to stop doing that, we should just delete this line and the code implementing it.

Reply:
This was another change that should never of got commited.

Reply:
Fixed in 7a8baf12b